packages/suite/src/components/wallet/KYCError/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ReactSVG } from 'react-svg';
+import styled from 'styled-components';
+import { Translation } from '@suite-components';
+import { resolveStaticPath } from '@trezor/utils';
+import { SavingsKYCCard } from '@wallet-views/coinmarket';
+
+const StyledCard = styled(SavingsKYCCard)`
+    background: rgba(239, 65, 65, 0.1);
+    color: ${props => props.theme.BG_RED};
+`;
+const Icon = styled.div`
+    margin-right: 17px;
+`;
+const Text = styled.div``;
+const Header = styled.div`
+    font-size: 16px;
+    line-height: 28px;
+`;
+const Description = styled.div`
+    font-size: 14px;
+    line-height: 18px;
+`;
+
+const KYCError = () => (
+    <StyledCard>
+        <Icon>
+            <ReactSVG src={resolveStaticPath('images/svg/warning.svg')} />
```

Comment:
Why `ReactSVG` is used here and not regular `img` as it is used almost everywhere else?